pkg/crc/cluster/cluster.go

```diff
@@ -77,3 +92,62 @@ func GetRootPartitionUsage(sshRunner *ssh.SSHRunner) (int64, int64, error) {
 	}
 	return diskSize, diskUsage, nil
 }
+
+func AddPullSecret(sshRunner *ssh.SSHRunner, oc oc.OcConfig, pullSec string) error {
+	if err := addPullSecretToInstanceDisk(sshRunner, pullSec); err != nil {
+		return err
+	}
+
+	base64OfPullSec := base64.StdEncoding.EncodeToString([]byte(pullSec))
+	cmdArgs := []string{"patch", "secret", "pull-secret", "-p",
```

Comment:
This used to be `oc replace`, not sure why you switched to `oc patch`?

Reply:
@cfergeau `oc replace` need whole file to be available but since resource already available we just need to use patch to update it.